### DISCUSSION TITLE
Aqara temperature sensor "WSDCGQ11LM" reporting setup

### DISCUSSION THREAD

**Discussion by pol-henri** [Koenkk/zigbee2mqtt#7085] [Devices]
Hello I would like to increase the precision of my aqara "WSDCGQ11LM" sensor, indeed it sends the information too slowly, I would like to have an update every 0.2 degrees, currently sometimes it is 1 degree, sometimes 0.5, can you please me? indicates the procedure to follow, via the web page and its reporting menu I cannot find the right setup
thank you in advance to all

**Comment by Koenkk** [comment]
The WSDCGQ11LM doesn't support reporting, so this is not possible with this device.

**Reply by pol-henri** [reply]
Thanks for your reply, there is another way to receive hight frequency update maebe globally ?

**Reply by Koenkk** [reply]
AFAIK no

**Reply by Jaguaza** [reply]
> El WSDCGQ11LM no admite informes, por lo que esto no es posible con este dispositivo.

@Koenkk Good morning, I have this problem with these sensors I force the update after 30 minutes and I observe that if there is a considerable temperature rise.

Do you know of any device that your update is higher, it is to control the temperature of the rooms for heating I can not afford to have the heating. Thank you.

![image](https://user-images.githubusercontent.com/46264854/149917446-f19d6a65-4079-4991-bc16-c153600b6525.png)

**Reply by k-zakhariy** [reply]
@Jaguaza could you share how did you setup to force update ? which mqtt command did you send ?

**Reply by Jaguaza** [reply]
I changed to other sensors, as I was not getting good refreshment with these.

**Reply by Machete3000** [reply]
Yep, the "WSDCGQ11LM" sensors  are pretty useless for heating / cooling control.

![image](https://github.com/Koenkk/zigbee2mqtt/assets/14955325/197b1b4c-3b58-47c9-b767-e0658de616d3)

Blue: Aqara WSDCGQ11LM
Purple: AcuRite 06002M
